crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -787,6 +798,75 @@ export.validate_seed
     # => []
 end
 
+#! Adds the specified asset to the account vault.
+#!
+#! Inputs:  [ASSET]
+#! Outputs: [ASSET']
+#!
+#! Where:
+#! - ASSET is the asset that is added to the vault.
+#! - ASSET' final asset in the account vault defined as follows:
+#!   - If ASSET is a non-fungible asset, then ASSET' is the same as ASSET.
+#!   - If ASSET is a fungible asset, then ASSET' is the total fungible asset in the account vault
+#!     after ASSET was added to it.
+#!
+#! Panics if:
+#! - the asset is not valid.
+#! - the total value of two fungible assets is greater than or equal to 2^63.
```

Comment:
```suggestion
#! - the total value of the fungible asset is greater than or equal to 2^63 after the new asset was added.
```

Nit: This might be clearer (should also update the corresponding panic info in `api.masm` then).